pkgs/desktops/gnome-3/core/gnome-desktop/bubblewrap-paths.patch

```diff
@@ -0,0 +1,19 @@
+--- a/libgnome-desktop/gnome-desktop-thumbnail-script.c
++++ b/libgnome-desktop/gnome-desktop-thumbnail-script.c
+@@ -504,14 +504,10 @@
+   g_return_val_if_fail (script->s_infile != NULL, FALSE);
+ 
+   add_args (array,
+-	    "bwrap",
+-	    "--ro-bind", "/usr", "/usr",
+-	    "--ro-bind", "/lib", "/lib",
+-	    "--ro-bind", "/lib64", "/lib64",
++	    "@BUBBLEWRAP_BIN@",
++      "--ro-bind", "/nix", "/nix",
```

Comment:
Is binding `/nix` enough or should I rather bind `/nix/store`?

Reply:
Since `/nix/var` is not needed, we should rather bind `/nix/store`.